test/integration.js

```diff
@@ -0,0 +1,112 @@
+import {spawn} from 'child_process'
+
+import test from 'ava'
+
+import pify from 'pify'
+
+import m from '..'
+
+test('should work with a single pid', async t => {
+  const pid = process.pid
+
+  const result = await pify(m)(pid)
+
+  t.log(result)
+
+  t.is(typeof result, 'object')
+
+  t.is(typeof result, 'object', 'result')
+  t.is(typeof result.cpu, 'number', 'cpu')
+  t.false(isNaN(result.cpu), 'cpu')
+  t.is(typeof result.memory, 'number', 'memory')
+  t.false(isNaN(result.memory), 'memory')
+  t.is(typeof result.ppid, 'number', 'ppid')
+  t.false(isNaN(result.ppid), 'ppid')
+  t.is(typeof result.pid, 'number', 'pid')
+  t.false(isNaN(result.pid), 'pid')
+  t.is(typeof result.elapsed, 'number', 'elapsed')
+  t.false(isNaN(result.elapsed), 'elapsed')
+  t.is(typeof result.timestamp, 'number', 'timestamp')
+  t.false(isNaN(result.timestamp), 'timestamp')
+})
```

Comment:
Now you make me learn a new test framework before all I used was:

```
t.ok()
t.equa()
t.deepEqual()
```

Again what's the benefit of using promises, await all this? Ok it's fancy but I wanted to keep this stuff workable on node 4 with no babel or fancy stuff...

Reply:
Why to bother about compatibility if this is handled by ava internally?
Tests are meant to be easy to write and read.